.github/workflows/build-vscode-extension.yml

```diff
@@ -31,6 +31,21 @@ jobs:
         working-directory: vscode/microsoft-kiota
       - run: npm run lint
         working-directory: vscode/microsoft-kiota
+      - name: run tests(linux)
+        if: runner.os == 'Linux'
+        run: xvfb-run -a npm run test-with-coverage
+        working-directory: vscode/microsoft-kiota
+      - name: run tests
+        if: runner.os != 'Linux'
+        run: npm run test-with-coverage
+        working-directory: vscode/microsoft-kiota
+      - name: Run sonar cloud analysis
+        uses: SonarSource/sonarcloud-github-action@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # Needed to get PR information, if any
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        with:
+          projectBaseDir: vscode/microsoft-kiota
       - run: npm run package
         working-directory: vscode/microsoft-kiota
       - run: npm i -g @vscode/vsce
```

Comment:
just leaving a comment to make sure we don't disable sonarcloud ingestion for CS files (deleted right now for testing purposes)